.gitignore

```
@@ -22,3 +22,117 @@
+.next/server/pages/o/[id]/roles.js
+.next/server/pages/o/[id]/users.js
+.next/static/chunks/_error.js
+.next/static/chunks/main.js
+.next/static/chunks/polyfills.js
+.next/static/chunks/react-refresh.js
+.next/static/chunks/webpack.js
+.next/static/chunks/fallback/amp.js
+.next/static/chunks/fallback/main.js
+.next/static/chunks/fallback/react-refresh.js
+.next/static/chunks/fallback/webpack.js
+.next/static/chunks/fallback/pages/_app.js
+.next/static/chunks/fallback/pages/_error.js
+.next/static/chunks/pages/_app.js
+.next/static/chunks/pages/_error.js
+.next/static/chunks/pages/404.js
+.next/static/chunks/pages/index.js
+.next/static/chunks/pages/o/[id].js
+.next/static/chunks/pages/o/moveOrg.js
+.next/static/chunks/pages/o/[id]/groups.js
+.next/static/chunks/pages/o/[id]/loginConnections.js
+.next/static/chunks/pages/o/[id]/profile.js
+.next/static/chunks/pages/o/[id]/resources.js
+.next/static/chunks/pages/o/[id]/roles.js
+.next/static/chunks/pages/o/[id]/users.js
+.next/static/development/_buildManifest.js
+.next/static/development/_ssgManifest.js
+.next/static/media/404.2ad6e2a6.svg
+.next/static/media/asgardeo-logo-transparent.23f2c848.png
+.next/static/media/docs.3a6775b3.png
+.next/static/media/enterprise.b3b34bc1.svg
+.next/static/media/github.ebcf7a0b.png
+.next/static/media/google.d57dfba8.svg
+.next/static/media/next.3baf90a6.svg
+.next/static/media/user.e42f057e.png
+.next/static/webpack/1ac1ecd37b519a34.webpack.hot-update.json
+.next/static/webpack/3a73cb774d97f904.webpack.hot-update.json
+.next/static/webpack/5fb6c4c2029c0383.webpack.hot-update.json
+.next/static/webpack/6aa26757cf7c3be2.webpack.hot-update.json
+.next/static/webpack/39ddc10dc4311d43.webpack.hot-update.json
+.next/static/webpack/99f38f889460738f.webpack.hot-update.json
+.next/static/webpack/240bfa5282cd1e25.webpack.hot-update.json
+.next/static/webpack/309e1598dc91697a.webpack.hot-update.json
+.next/static/webpack/4150749abad70e97.webpack.hot-update.json
+.next/static/webpack/a9524cd437a97aba.webpack.hot-update.json
+.next/static/webpack/e5fc2e09be9d7be7.webpack.hot-update.json
+.next/static/webpack/f0458b23b83ee00e.webpack.hot-update.json
+.next/static/webpack/webpack.1ac1ecd37b519a34.hot-update.js
+.next/static/webpack/webpack.3a73cb774d97f904.hot-update.js
+.next/static/webpack/webpack.5fb6c4c2029c0383.hot-update.js
+.next/static/webpack/webpack.6aa26757cf7c3be2.hot-update.js
+.next/static/webpack/webpack.39ddc10dc4311d43.hot-update.js
+.next/static/webpack/webpack.99f38f889460738f.hot-update.js
+.next/static/webpack/webpack.240bfa5282cd1e25.hot-update.js
+.next/static/webpack/webpack.309e1598dc91697a.hot-update.js
+.next/static/webpack/webpack.4150749abad70e97.hot-update.js
+.next/static/webpack/webpack.a9524cd437a97aba.hot-update.js
+.next/static/webpack/webpack.e5fc2e09be9d7be7.hot-update.js
+.next/static/webpack/webpack.f0458b23b83ee00e.hot-update.js
+.next/static/webpack/pages/o/[id]/resources.e5fc2e09be9d7be7.hot-update.js
```

We can use the below notation to ignore all files and folders in the .next folder.

```suggestion
.next/**/*
```

---

Fixed